core/tests/test_utils.py

```diff
@@ -2562,10 +2562,17 @@ def traverse_schema_and_assign_content_ids(value, schema, contentId):
         state.update_next_content_id_index(next_content_id_index_dict['value'])
 
     def save_new_valid_exploration(
```

Comment:
I annotated only these 2 functions because these 2 occur in test file too frequently!